### DISCUSSION TITLE
Uncaught ReferenceError: pgFlatpickr is not defined

### DISCUSSION THREAD

**Discussion by ewkilian** [Power-Components/livewire-powergrid#1063] [Q&A]
Hi,
I install PowerGrid on my existing Laravel projet.

But I have this error :
"Uncaught ReferenceError: pgFlatpickr is not defined"

I have try a lot of manipulation, search on the web, but I find the solution 

You can see the dependance : 
       "php": "^8.1",
        "dompdf/dompdf": "^2.0",
        "guzzlehttp/guzzle": "^7.2",
        "laravel/framework": "^10.0",
        "laravel/sanctum": "^3.2",
        "laravel/tinker": "^2.8",
        "laravel/ui": "^4.2",
        "laraveldaily/laravel-charts": "^0.1.29",
        "livewire/livewire": "^2.11",
        "maatwebsite/excel": "^3.1",
        "nuovo/spreadsheet-reader": "^0.5.11",
        "power-components/livewire-powergrid": "^4.5",
        "spatie/laravel-medialibrary": "^10.7"

Have you a idea to help me ? you need more information ?

Thanks you a lot

**Comment by ewkilian** [comment]
Hi,
I'm block, don't find a solution. Have you a idea top help me ?
Thank you a lot
Bye

**Comment by luanfreitasdev** [comment]
Hello @ewkilian, Did you follow these steps?

https://v4.livewire-powergrid.com/get-started/configure.html#_1-import-js-and-css
